modules/ossm-performing-inplace-update.adoc

```diff
@@ -0,0 +1,40 @@
+:_mod-docs-content-type: Procedure
+[id="performing-inplace-updtate"]
+= Performing InPlace update
+:context: ossm-performing-inplace-update
+
+When updating {istio} using the `InPlace` strategy, you can only increment by one minor version at a time. If you want to update by more than one minor version, then you must increment the version and restart the workloads after each update. This ensures that the sidecar version is compatible with the control plane version. After all the workloads are restarted the update process is complete.
+
+.Prerequisites
+
+* You are logged in to {product-title} as `cluster-admin`.
+* You have installed the {SMProductName} Operator, and deployed {istio}.
+
+.Procedure
+
+1. Change the version in the {istio} resource:
```

Comment:
you probably should remove the explicit list numbering here

Reply:
done